### PR TITLE
Add decision regarding support of PyCon NA.

### LINKED ISSUE #38: PyCon Namibia Request
Dear Directors of the UK Python Association,

The sixth successive edition of PyCon Namibia https://na.pycon.org/ takes place 18-20 February 2020, at the NamPower Convention Centre in Windhoek.

Once again, the organisers would like to request the support of the UK Python Association as a financial sponsor. 

This year’s sponsorship page is at https://na.pycon.org/sponsorship/. A PDF with the same information is attached.

The UK Python community has had a long association with PyCon Namibia and has supported us since the beginning. Last year the UK Python Association sponsored our event, and the activities and outcomes it helped support are detailed in our report: https://na.pycon.org/2019/.

PyCon Namibia has been an influential event in the evolution of the Python movement across Africa, and we aim to continue that work. The conference and its organisers past and present have been key to the establishment of other African Python events, including PyCons in Nigeria, Zimbabwe and Ghana, PyCon Africa, and the forthcoming DjangoCon Africa. 

We will continue this work, but in 2020 our key focus will be on education and the Namibian Python community.

As always, PyCon Namibia will publish a full conference report after the event. The event is covered by our well-publicised code of conduct https://na.pycon.org/code-of-conduct/

The UK Python Association's support will help us cover basic costs of the conference: venue hire, catering and our support for attendees whose financial means are limited.

We would like to request a grant from the UK Python Association once more in the amount of US$2000.

With our sincere thanks for your consideration,

Jessica Upani and the PyCon Namibia 2020 organising team

### PR DESCRIPTION
**Pull request by drvinceknight**
Closes #38

### CODE REVIEW

File: source/decisions/pyconna2020.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
+Response to PyCon Namibia Sponsorhip Request
```
Comment (meatballs) [inline source/decisions/pyconna2020.rst]: typo: Sponsorship

File: source/decisions/pyconna2020.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+============================================
+
+Request
+-------
+
+The original request came via email, the relevant excerpts are:
```
Comment (meatballs) [inline source/decisions/pyconna2020.rst]: niggle: replace the comma with a period and start a new sentence.

**Comment by doismellburning** [comment]
@drvinceknight Thank you for doing this! I hope you don't mind - I took the liberty of fiddling with the formatting slightly while I was there

**Comment by drvinceknight** [comment]
> @drvinceknight Thank you for doing this! I hope you don't mind - I took the liberty of fiddling with the formatting slightly and adding a little context while I was there

No problem at all, thank you very much for taking the time.

**Pull request merged**